juno/assetlist.json

```diff
@@ -1427,7 +1427,50 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.svg"
+             }
+  },
+  {
+    "description": "Propose. Vote. Build.",
+    "type_asset": "cw20",
+    "address": "juno12mcwmd6wqhledkjsurlfqtc8j0pedvxlcxw3gs4kh2qf808ehehsen8nmw",
+    "denom_units": [
```

Comment:
Looks like this is trying to add a duplicate entry to yfd, which is already added above.